documentation/Demo 1/Requirments.md

```diff
@@ -0,0 +1,79 @@
+# Red Hat Beyond 07
+
+Statement of Work (SOW)
+***
+
+
+## Milestones
+
+### Demo 1 (March 13th): Project selection and setting up a DEV environment
```

Comment:
and here link to this part like:
https://github.com/redhat-beyond/documents/blob/main/Beyond07-SOW.md#demo-1-march-13th-project-selection-and-setting-up-a-dev-environment

Reply:
done